<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_Admin</class>
 <widget class="QDialog" name="Login_Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-image: url(:/pictures/3407709_m.jpg);
	font-family:century gothic;
	font-size:24px;
}

QFrame
{
	background:black;
	border-radius:25;
}

QPushButton
{
	background:blue;
    border-radius: 20;
}

QToolButton
{
	background:blue;
    border-radius: 60;
}
QLabel
{
color:white;
}

QPushButton
{
   	color:white;
    border-radius: 15;
}

QPushButton:hover
{
   	color:white;
    border-radius: 15;
	background:red;
}

QLineEdit
{
	background: transparent;
	border: none;	
	color: #717072;
	border-bottom:1px solid #717072;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>521</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>LOGIN HERE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loginAdmin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Nom d'utilisateur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Messagelog">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
color:red;
font: 14px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/pictures/R29ff80e1b872c3a34502081bb24ddc77.png);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/pictures/pictures/R29ff80e1b872c3a34502081bb24ddc77.png</normaloff>:/pictures/pictures/R29ff80e1b872c3a34502081bb24ddc77.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_retourMenu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>510</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
background:blue;
border-radius:6;
font:15px;
}
QPushButton:hover
{
background:red;
}</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
